src/catalog-protos/src/serialization.rs

```diff
@@ -566,18 +566,6 @@ impl RustType<crate::objects::EpochMillis> for u64 {
     }
 }
 
-impl RustType<crate::objects::Timestamp> for Timestamp {
```

Comment:
We're not using this implementation. Should remove the `Timestamp` type from the proto, but that'd require a migration.